source/js/components/highlights/highlights.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export default class Highlights extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+
+  render() {
+    // Limit to 2 additional highlights
+    let highlights = this.props.data.slice(1, 3).map((item, index, array) => {
```

Comment:
we should type check `this.props.data` and probably also give it a default value `[]`